### ISSUE TITLE
Errors are no longer re-raised in ActiveJob 4.2

### ISSUE THREAD

**Issue by knapo** [getsentry/raven-ruby#726]
https://github.com/getsentry/raven-ruby/commit/18fafd655a8a85540602419234e22b32949d9400 broke active job 4.2 compatibility and errors are no longer re-raised:

```ruby
[7] pry(main)> class MyJob  < ActiveJob::Base
[7] pry(main)*   def perform(p)  
[7] pry(main)*     fail 'I am an error'    
[7] pry(main)*   end  
[7] pry(main)* end  
=> :perform
[8] pry(main)> T1.perform_now('hello')
Performing T1 from Sidekiq(default) with arguments: "hello"
Performed T1 from Sidekiq(default) in 0.97ms
=> nil
```

**Comment by nateberkopec** [comment]
Which ActiveJob adapter are you using?

**Comment by knapo** [comment]
sidekiq

**Comment by nateberkopec** [comment]
Fixed in 5b02ad4ff2a7, release OTW
